vnext/PropertySheets/React.Cpp.props

```diff
@@ -95,6 +95,9 @@
         BOOST_SYSTEM_SOURCE;
         GTEST_HAS_RTTI=0;
         WIN32_LEAN_AND_MEAN;
+        %(PreprocessorDefinitions)
+      </PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(NoWinRtLeanAndMean)' != 'true'">
```

Comment:
Consider changing this to a property to enable lean and mean that's set to true by default.

Reply:
You're right. I was just lazy as I didn't want to wrestle with msbuilds model too much... I'll take care of it on monday.